### PR TITLE
milestone 5: styling 

### PR DESCRIPTION
**Pull request by bronnie313**
## Bookstore; Styling  
-  Added missing jsx elements to implement styling 
- added styling

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -28,14 +28,14 @@ const Book = ({
...
           <div className="updates">
-            <p>Current Chapter</p>
-            <p>Chapter 12</p>
-            <button type="button"> UPDATE PROGRESS</button>
+            <p className="current-chapter">Current Chapter</p>
+            <p className="chapter">Chapter 12</p>
+            <button className="update-progress" type="button"> UPDATE PROGRESS</button>
           </div>
         </div>
       </div>
```
Comment (muneebulrehman) [inline src/components/Book.js:41]: When adding a book, it doesn't show up in the list of books. I have to reload to see the book appear in the list. Kindly fix this issue. 

https://github.com/bronnie313/bookstore/assets/33268927/8c1f0472-5b04-4fc5-a9a3-357f3ee7b09a

File: src/components/styles/Header.css [hunk 2]
```diff
@@ -4,12 +4,13 @@ nav {
   padding: 1.5rem 5rem;
   border-bottom: 1px solid lightgray;
   align-items: center;
+  background-color: white;
 }
```
Comment (muneebulrehman) [inline src/components/styles/Header.css:8]: Your navbar is the not matching the zeplin design provided to you. Requirement of this milestone is to replicate the zeplin design exactly as it is. Kindly do so and then submit for a re-review.
| Your design  | Figma design |
| ------------- | ------------- |
| image | image |

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @bronnie313 , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Great job trying to replicate the zeplin design.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/Header.css (hunk 2 above)
Comment (bronnie313) [reply src/components/styles/Header.css:8]: thanks for the review

**Review by fobadara — APPROVED** [review]
Hi @bronnie313, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Clean code

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**